ssh_fr.md

```diff
@@ -1,26 +1,65 @@
 # SSH
-Le **SSH** permet de commander à distance son serveur en ligne de commande (CLI).
 
-#### Pour se connecter à son serveur
+## Qu'est-ce que SSH ?
+
+**SSH** est un accronyme pour Secure Shell, et désigne un protocole qui permet de contrôler à distance une machine via la ligne de commande (CLI). C'est aussi une commande disponible de base sur Linux et Mac OS / OSX. Sous Windows, maleureusement il vous faudra utiliser le logiciel [PuTTy](http://www.fastcomet.com/tutorials/getting-started/putty).
+
+## Pendant l'installation de YunoHost
+
+#### Trouver son IP
+
+Si vous installez YunoHost sur un VPS, votre fournisseur devrait vous avoir communiqué l'adresse IP de votre serveur. 
+
+Si vous installez un serveur à la maison (par ex. sur Raspberry Pi ou OLinuXino), il vous faut trouver l'IP qui a été attribuée à votre carte après que vous l'ayez connecté à votre box internet / routeur. Il y a plusieurs façon de faire ça :
+
+- ouvrez un terminal et tapez `sudo arp-scan --local` pour lister les IP des machines sur le réseau local ;
+- utilisez l'interface de votre box internet pour lister les clients DHCP connectés ;
+- branchez un écran sur votre serveur, loggez-vous et tapez `ifconfig`.
```

Comment:
Add other options listed on english version:

"look into the logs of your rooter"
"you can also uses nmap -sP $local-subnet/24 to see all conected ips and test them [advanced]"